### PR TITLE
SAK-41218 Add a message with maximum file size for content review fil…

### PR DESCRIPTION
**Pull request by josecebe**
…e on teacher and student view

### CODE REVIEW

File: assignment/tool/src/webapp/vm/assignment/chef_assignments_student_preview_submission.vm [hunk 1]
```diff
@@ -149,8 +149,15 @@
 				<img alt="$reviewIndicator" title="$reviewIndicator" src="/library/image/silk/rosette.png" class="assignmentRosette"/>
 				<em>
 					$plagiarismNote
-					#if ($plagiarismFileTypes)
-						$plagiarismFileTypes
+					#if ($plagiarismFileTypes || $plagiarismFileTypes)
```
Comment (bgarciaentornos) [inline assignment/tool/src/webapp/vm/assignment/chef_assignments_student_preview_submission.vm]: Same variable is twice, guess it should be plagiarismFileSize

**Comment by josecebe** [comment]
Done @bgarciaentornos @bjones86

File: assignment/tool/src/webapp/vm/assignment/chef_assignments_student_preview_submission.vm (hunk 1 above)
Comment (bjones86) [reply assignment/tool/src/webapp/vm/assignment/chef_assignments_student_preview_submission.vm]: @josecebe, I'm still seeing the same variable twice in this conditional. As @bgarciaentornos mentioned, one of them should probably be `plagiarismFileSize`

File: assignment/tool/src/webapp/vm/assignment/chef_assignments_student_view_submission.vm [hunk 2]
```diff
@@ -596,8 +596,15 @@ $(document).ready(function(){
 									<img alt="$reviewIndicator" title="$reviewIndicator" src="/library/image/silk/rosette.png" class="assignmentRosette"/>
 									<em>
 										$plagiarismNote
-										#if ($plagiarismFileTypes)
-											$plagiarismFileTypes
+										#if ($plagiarismFileTypes || $plagiarismFileTypes)
```
Comment (bjones86) [inline assignment/tool/src/webapp/vm/assignment/chef_assignments_student_view_submission.vm]: Same here

File: assignment/tool/src/webapp/vm/assignment/chef_assignments_student_view_submission.vm [hunk 3]
```diff
@@ -596,8 +596,15 @@ $(document).ready(function(){
...
+											<ul>
+												#if ($plagiarismFileTypes)
```
Comment (bjones86) [inline assignment/tool/src/webapp/vm/assignment/chef_assignments_student_view_submission.vm]: And here

File: assignment/tool/src/webapp/vm/assignment/chef_assignments_student_view_submission.vm [hunk 4]
```diff
@@ -686,8 +693,15 @@ $(document).ready(function(){
 							<img alt="$reviewIndicator" title="$reviewIndicator" src="/library/image/silk/rosette.png" class="assignmentRosette"/>
 							<em>
 								$plagiarismNote
-								#if ($plagiarismFileTypes)
-									$plagiarismFileTypes
+								#if ($plagiarismFileTypes || $plagiarismFileTypes)
```
Comment (bjones86) [inline assignment/tool/src/webapp/vm/assignment/chef_assignments_student_view_submission.vm]: And here

File: assignment/tool/src/java/org/sakaiproject/assignment/tool/AssignmentAction.java [hunk 5]
```diff
@@ -1466,6 +1467,14 @@ private String build_student_view_submission_context(VelocityPortlet portlet, Co
                 		state.setAttribute("plagiarismFileTypes", rb.getFormattedMessage("gen.onlythefoll", getContentReviewAcceptedFileTypesMessage())); 
                     context.put("plagiarismFileTypes", state.getAttribute("plagiarismFileTypes"));
 
+                    String reviewServiceName = contentReviewService.getServiceName();
+                    if ("Urkund".equals(reviewServiceName)) {
```
Comment (bjones86) [inline assignment/tool/src/java/org/sakaiproject/assignment/tool/AssignmentAction.java]: I'm not sure why this is specific to just the Urkund service. I'm also not a fan of hardcoding this string.

File: assignment/tool/src/webapp/vm/assignment/chef_assignments_student_view_submission.vm (hunk 4 above)
Comment (josecebe) [reply assignment/tool/src/webapp/vm/assignment/chef_assignments_student_view_submission.vm]: Thanks, I did it on the wrong branch and not fixed it in here.

File: assignment/tool/src/java/org/sakaiproject/assignment/tool/AssignmentAction.java (hunk 5 above)
Comment (josecebe) [reply assignment/tool/src/java/org/sakaiproject/assignment/tool/AssignmentAction.java]: As far as I know, Urkund is the only one that allows to modify the maximum file size. So it needs to be "filtered" by the content review provider. In other parts of the code also makes this making a comparison by a hardcoded string. I'm going to change this code fragment a bit.

**Comment by bgarciaentornos** [comment]
@bjones86 Can you please take a look to this? It seems the requested changes have been done. Thanks!

**Comment by bjones86** [comment]
Looks good, thanks @josecebe

**Pull request merged**